packages/wrapped-keys-lit-actions/src/lib/common/internal/getDecryptedKey.js

```diff
@@ -0,0 +1,20 @@
+/* global Lit */
+
+export async function getDecryptedKey({
```

Comment:
nit: can this be renamed to `getDecryptKeyToSingleNode`. Makes it easier to understand from a caller perspective that it can not be chosen and get `undefined`

Reply:
Good idea :+1: <3